### PR TITLE
Mileston 1/feat scrum 57 user flight tracking testing

### PR DESCRIPTION
**Pull request by Rawan-M-Said**

### CODE REVIEW

File: backend/pom.xml [hunk 1]
```diff
@@ -57,6 +57,17 @@
...
 		</dependency>
+		<dependency>
+			<groupId>org.mockito</groupId>
+			<artifactId>mockito-all</artifactId>
+			<version>2.0.2-beta</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.assertj</groupId>
+			<artifactId>assertj-core</artifactId>
```
Comment (OmniaTarek18) [inline backend/pom.xml]: no need to add assertj or mockito dependencies as they already exist in test dependency

**Comment by OmniaTarek18** [comment]
I prefer to rename the tests of Repos to Repository to keep consistency

File: backend/pom.xml (hunk 1 above)
Comment (Rawan-M-Said) [reply backend/pom.xml]: I modified it.

**Comment by Rawan-M-Said** [comment]
> I prefer to rename the tests of Repos to Repository to keep consistency

Updated the names, is there anything else.

**Pull request merged**